queries/typescript/highlights.scm

```diff
@@ -136,6 +136,9 @@
 (arrow_function
   parameter: (identifier) @parameter)
 
+;; global declaration
+(ambient_declaration "global" @namespace)
```

Comment:
```suggestion
(ambient_declaration "global" @storageclass)
```

Reply:
Thanks for the suggestion.
I'm certainly not a TypeScript expert, but isn't `declare global {}` symmetrical to `declare namespace SomeClass {}`?

Reply:
Me neither - how about `@keyword` then?

Reply:
It could be, but I picked `@namespace` because of semantic tokens from `tsserver`.
I might be wrong and this is a semantic token from CoC.nvim.
Forgive me for my Neovim setup, I haven't finished my migration to Neovim LSP yet. 😆

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/4299733/a14cf741-8421-49cd-8872-8b9cfe83055e)

Reply:
`@namespace` seems fine to me.